EssentialCSharp.Web/wwwroot/css/styles.css

```diff
@@ -485,6 +485,7 @@ details > summary::-webkit-details-marker {
   padding-top: 15px;
   padding-right: 18px;
   padding-left: 15px;
+  scrollbar-gutter: stable;
```

Comment:
Thanks @COsborn2 for making this change

I've reviewed the updates, and I'll :shipit: now.